### DISCUSSION TITLE
"use_branching_in_lp" parameter in v9.9

### DISCUSSION THREAD

**Discussion by IgnaceBleukx** [google/or-tools#4139] [CP-SAT questions] [answered]
Hi, 

I am doing some hyperparameter tuning for the CP-SAT solver for some of the models I got.
Version 9.8 had the parameter `use_branching_in_lp` Boolean parameter, but it seems to be removed in v9.9.
Is there a way to still turn on/off the branching? Or is the parameter just renamed?

Kind regards,
Ignace

**Comment by lperron** [comment, marked as answer]
It was removed.
We had zero worker using it.

I recommend using multiple workers as it does a good job at having diverse strategies.
Otherwise, you can have a look at https://github.com/google/or-tools/blob/stable/ortools/sat/cp_model_search.cc#L649
